features/received_command.feature

```diff
@@ -20,6 +20,23 @@ Feature:  received_command
     When item "Alarm_Mode" state is changed to "7"
     Then It should log 'Item received command: 7' within 5 seconds
 
+  Scenario: Rule supports helper predicates for named commands
+    Given a deployed rule:
+      """
+      rule 'Execute rule when item received command' do
+        received_command Alarm_Mode
+        run do |event|
+          %i[refresh? on? off? increase? decrease? up? down? stop? move? play? pause? rewind? fastforward? next? previous?].each do |pred|
```

Comment:
I can't figure out quickly by looking at it what we are testing here? 
While not as fast, often you do this sort of multi-test with a table in cucumber so you know exactly which scenarios aren't behaving.

Reply:
basically testing that the ItemCommandEvent responds to all these methods. I'm not that worried about testing each one individually will return `true` for their corresponding command being sent, but I do want to make sure that they all exist. so the scenario tests that one of them returns true, but the rule executes the code for all of them, and would fail completely if it doesn't exist.